README.md

```diff
@@ -22,4 +23,4 @@ The Substrate Recipes are [GPL 3.0 Licensed](LICENSE) It is open source and [ope
 
 ## Using Recipes in External Projects
 
-The pallets and runtimes provided here are tested and ready to be used in other Substrate-based blockchains. The big caveat is that you must use the same upstream Substrate version throughout the project. The recipes currently use Substrate@`v2.0.0-alpha.7`.
+The pallets and runtimes provided here are tested and ready to be used in other Substrate-based blockchains. The big caveat is that you must use the same upstream Substrate version throughout the project. The recipes currently use Substrate@`7e9a2ae`.
```

Comment:
By specifying the Substrate version here, so we have to update this line every time we upgrade to a new substrate version?This may be easily neglected. I think dev could just inspect `Cargo.toml` to figure this if they plan to integrate our pallets to their projects.

Reply:
Agree. I've removed that from the README